azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/autoconfigure/initializer/SpringBootTelemetryInitializer.java

```diff
@@ -1,4 +1,4 @@
-package com.microsoft.applicationinsights.boot.initializer;
```

Comment:
This comment applies to all relevant files: I don't get the package rename. If it were me, I'd go with `com.microsoft.applicationinsights.web.spring.boot.*`, modeling after the other spring related classes we have.